e2e/page-objects/profilePage.ts

```diff
@@ -67,6 +67,27 @@ export class ProfilePage extends HelperBase {
     return fullNameValue;
   }
 
+  async getOsmEditorValue() {
+    const osmEditorValue = await this.page
+      .getByText('Default OpenStreetMap editor (beta)')
+      .locator('..')
+      .locator('span')
+      .textContent();
+    return osmEditorValue;
+  }
+
+  async setOsmEditorValue(osmEditorValue: string) {
```

Comment:
please, provide descriptions to methods the same way it is done for other methods in page objects. It is needed for better readability of the code for people not familiar with our system